xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -47,6 +90,30 @@
         <remarks>When using this constructor, the format of the resulting archive is <see cref="F:System.Formats.Tar.TarFormat.Pax" />.</remarks>
       </Docs>
     </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public TarWriter (System.IO.Stream archiveStream, System.Formats.Tar.TarEntryFormat format = System.Formats.Tar.TarEntryFormat.Pax, bool leaveOpen = false);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.IO.Stream archiveStream, valuetype System.Formats.Tar.TarEntryFormat format, bool leaveOpen) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Formats.Tar.TarWriter.#ctor(System.IO.Stream,System.Formats.Tar.TarEntryFormat,System.Boolean)" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New (archiveStream As Stream, Optional format As TarEntryFormat = System.Formats.Tar.TarEntryFormat.Pax, Optional leaveOpen As Boolean = false)" />
+      <MemberSignature Language="F#" Value="new System.Formats.Tar.TarWriter : System.IO.Stream * System.Formats.Tar.TarEntryFormat * bool -&gt; System.Formats.Tar.TarWriter" Usage="new System.Formats.Tar.TarWriter (archiveStream, format, leaveOpen)" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Formats.Tar</AssemblyName>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="archiveStream" Type="System.IO.Stream" />
+        <Parameter Name="format" Type="System.Formats.Tar.TarEntryFormat" />
+        <Parameter Name="leaveOpen" Type="System.Boolean" />
+      </Parameters>
+      <Docs>
+        <param name="archiveStream">To be added.</param>
+        <param name="format">To be added.</param>
+        <param name="leaveOpen">To be added.</param>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
     <Member MemberName=".ctor">
       <MemberSignature Language="C#" Value="public TarWriter (System.IO.Stream archiveStream, System.Formats.Tar.TarFormat archiveFormat, bool leaveOpen = false);" />
       <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.IO.Stream archiveStream, valuetype System.Formats.Tar.TarFormat archiveFormat, bool leaveOpen) cil managed" />
```

Comment:
I am unable to provide suggestions for lines 120-122, but they need similar changes:

```suggestion
      <MemberSignature Language="C#" Value="public TarWriter (System.IO.Stream archiveStream, System.Formats.Tar.TarEntryFormat archiveFormat, bool leaveOpen = false);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.IO.Stream archiveStream, valuetype System.Formats.Tar.TarEntryFormat archiveFormat, bool leaveOpen) cil managed" />
```

Reply:
> This seems to be a bug in mdoc: The `MemberSignature` of this method should've changed `TarFormat` to `TarEntryFormat`. Known issue?

The update constructor was added on line 93, but you're right, this one should have been deleted. @huangmin-ms  FYI.

Reply:
It would make sense to keep the _old_ constructor that takes a `TarFormat` only if the whole `TarFormat.xml` file was kept as well, so the reference to the old API wasn't lost.

Reply:
@gewarren if we delete the old constructor, the docs on the old constructor will also be lost.  Maybe we can try with another branch: move the docs from old constructors to the new one, delete docs on the old constructor and rerun the pipeline and see if it will be deleted.

Reply:
Thank you for following up on this Min!